MEK6800D1: early Motorola 6800 design evaluation board

68bit opened the pull request:
The MIKBUG monitor is working. The terminal support for reading and 'punching' tapes is not yet implemented, but perhaps would be better implemented in a TTY emulator. Perhaps this of some historical interest. The ROM for this one is available at: https://68bit.org/mekd1.zip

cuavas commented on src/mame/drivers/mekd1.cpp:
```diff
@@ -0,0 +1,462 @@
+	}
+
+	if (m_bit_rate_reset)
+	{
```
Please be consistent with brace style - this is Allman, while you have K&R in other places.

cuavas commented on src/mame/drivers/mekd1.cpp:
```diff
@@ -0,0 +1,462 @@
+	// Allocate timers
+	m_bit_rate_timer = timer_alloc(TIMER_BIT_RATE);
+	m_bit_rate_half_timer = timer_alloc(TIMER_BIT_RATE_HALF);
+}
```
Please register state members for save/load.  We want to get to a point where machines/devices can be expected to support save states by default.

68bit commented:
Thanks. Fixed the style issues, and added `save_item` entries for some of the states. Not up to speed with the state saving yet and intended to come back to that for the machine I am working on, and yes that would be really nice to have working.
